src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -394,9 +420,39 @@ protected override void OnHostStarted()
                 _secretManager.PurgeOldSecretsAsync(Instance.ScriptConfig.RootScriptPath, Instance.TraceWriter, Instance.Logger);
             }
 
+            if (Microsoft.Azure.WebJobs.Script.WebHost.Management.FunctionsSyncManager.IsSyncTriggersEnvironment(_settingsManager))
+            {
+                // create a onetime invocation timer
+                if (_syncTimer != null)
+                {
+                    // cancel any outstanding timer
+                    _syncTimer.Change(Timeout.Infinite, Timeout.Infinite);
```

Comment:
Do we need to dispose this timer as well?

Reply:
I do dispose it in the Dispose method